File: src/main/java/com/graqr/threshr/model/redsky/store/Miscellaneous.java
package com.graqr.threshr.model.redsky.store;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record Miscellaneous(@JsonProperty("google_cid") String googleCid) {}

File: src/main/kotlin/com/graqr/threshr/model/redsky/store/Miscellaneous.kt
package com.graqr.threshr.model.redsky.store

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class Miscellaneous(@param:JsonProperty("google_cid") val googleCid: String) {}
